gravity.sh

```diff
@@ -216,6 +216,22 @@ gravity_DownloadBlocklistFromUrl() {
 
   str="Status:"
   echo -ne "  ${INFO} ${str} Pending..."
+  # Determine if the domain is blocked by Pi-hole
+  if [ `dig $domain +short | grep 0.0.0.0 -c` -ge 1 ]; then
+    # If the domain is blocked by Pi-hole, use an alternate dns server to lookup the ip adres
+    ip=`dig @1.1.1.1 +short $domain`
```

Comment:
Not a fan of hardcoding `1.1.1.1` as the alternative lookup. Especially considering user will have already set their upstream preference. (`/etc/pihole/setupVars.conf`)

Reply:
I agree. Now using suggested method

Reply:
Done